cmd/node-termination-handler.go

```diff
@@ -320,11 +335,14 @@ func drainOrCordonIfNecessary(interruptionEventStore *interruptioneventstore.Sto
 				log.Err(err).Msgf("node '%s' not found in the cluster", nodeName)
 			} else {
 				log.Err(err).Msg("There was a problem while trying to cordon and drain the node")
+				metrics.NodeActionsInc("cordon-and-drain", nodeName, err)
```

Comment:
Added the `metrics` line here too so a metric is also created in the event of an error